annubes/task.py

```diff
@@ -243,30 +230,26 @@ def _build_trials_inputs(self) -> NDArray[np.float32]:
             (self._ntrials, len(self.time), self.n_inputs),
             dtype=np.float32,
         )
-        sel_value_in = np.full(  # TODO: needs a better name
-            (self._ntrials, self.n_inputs - 1),  # should not include start cue
-            0,
-            dtype=np.float32,
-        )
 
         for n in range(self._ntrials):
             for idx, m in enumerate(self.modalities):
                 if (self._modality_seq[n] != "catch") and (m in self._modality_seq[n]):
-                    sel_value_in[n, idx] = self._rng.choice(self.stim_intensities[1:], 1)
-                sel_value_in[n, idx] = self._rescale(sel_value_in[n, idx], self.rescaling_coeff)
-                x[n, self._phases["input"], idx] = sel_value_in[n, idx]
-                x[n, self._phases["fix_time"], idx] = self._rescale(
-                    self.fix_intensity,
-                    self.rescaling_coeff,
-                )
+                    value = self._rng.choice(self.stim_intensities, 1)
```

Comment:
when would the second part of that if statement not be true?

Reply:
Never. I removed it :)